pkg/auth/authz_handler_test.go

```diff
@@ -676,63 +629,76 @@ func downloadClusterKubeconfig(ctx context.Context, cli *client.AssistedInstall)
 }
 
 func getClusterInstallConfig(ctx context.Context, cli *client.AssistedInstall) error {
-	_, err := cli.Installer.GetClusterInstallConfig(
+	_, err := cli.Installer.V2GetClusterInstallConfig(
 		ctx,
-		&installer.GetClusterInstallConfigParams{
+		&installer.V2GetClusterInstallConfigParams{
 			ClusterID: strfmt.UUID(uuid.New().String()),
 		})
 	return err
 }
 
 func updateClusterInstallConfig(ctx context.Context, cli *client.AssistedInstall) error {
-	_, err := cli.Installer.UpdateClusterInstallConfig(
+	_, err := cli.Installer.V2UpdateClusterInstallConfig(
 		ctx,
-		&installer.UpdateClusterInstallConfigParams{
-			ClusterID: strfmt.UUID(uuid.New().String()),
+		&installer.V2UpdateClusterInstallConfigParams{
+			ClusterID:           strfmt.UUID(uuid.New().String()),
+			InstallConfigParams: `{"controlPlane": {"hyperthreading": "Disabled"}}`,
 		})
 	return err
 }
 
 func uploadClusterIngressCert(ctx context.Context, cli *client.AssistedInstall) error {
-	_, err := cli.Installer.UploadClusterIngressCert(
-		ctx,
-		&installer.UploadClusterIngressCertParams{
-			ClusterID: strfmt.UUID(uuid.New().String()),
+	ingressCa := "-----BEGIN CERTIFICATE-----\nMIIDozCCAougAwIBAgIULCOqWTF" +
+		"aEA8gNEmV+rb7h1v0r3EwDQYJKoZIhvcNAQELBQAwYTELMAkGA1UEBhMCaXMxCzAJBgNVBAgMAmRk" +
+		"MQswCQYDVQQHDAJkZDELMAkGA1UECgwCZGQxCzAJBgNVBAsMAmRkMQswCQYDVQQDDAJkZDERMA8GCSqGSIb3DQEJARYCZGQwHhcNMjAwNTI1MTYwNTAwWhcNMzA" +
+		"wNTIzMTYwNTAwWjBhMQswCQYDVQQGEwJpczELMAkGA1UECAwCZGQxCzAJBgNVBAcMAmRkMQswCQYDVQQKDAJkZDELMAkGA1UECwwCZGQxCzAJBgNVBAMMAmRkMREwDwYJKoZIh" +
+		"vcNAQkBFgJkZDCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAML63CXkBb+lvrJKfdfYBHLDYfuaC6exCSqASUAosJWWrfyDiDMUbmfs06PLKyv7N8efDhza74ov0EQJ" +
+		"NRhMNaCE+A0ceq6ZXmmMswUYFdLAy8K2VMz5mroBFX8sj5PWVr6rDJ2ckBaFKWBB8NFmiK7MTWSIF9n8M107/9a0QURCvThUYu+sguzbsLODFtXUxG5rtTVKBVcPZvEfRky2Tkt4AySFS" +
+		"mkO6Kf4sBd7MC4mKWZm7K8k7HrZYz2usSpbrEtYGtr6MmN9hci+/ITDPE291DFkzIcDCF493v/3T+7XsnmQajh6kuI+bjIaACfo8N+twEoJf/N1PmphAQdEiC0CAwEAAaNTMFEwHQYDVR0O" +
+		"BBYEFNvmSprQQ2HUUtPxs6UOuxq9lKKpMB8GA1UdIwQYMBaAFNvmSprQQ2HUUtPxs6UOuxq9lKKpMA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQELBQADggEBAJEWxnxtQV5IqPVRr2SM" +
+		"WNNxcJ7A/wyet39l5VhHjbrQGynk5WS80psn/riLUfIvtzYMWC0IR0pIMQuMDF5sNcKp4D8Xnrd+Bl/4/Iy/iTOoHlw+sPkKv+NL2XR3iO8bSDwjtjvd6L5NkUuzsRoSkQCG2fHASqqgFoyV9Ld" +
+		"RsQa1w9ZGebtEWLuGsrJtR7gaFECqJnDbb0aPUMixmpMHID8kt154TrLhVFmMEqGGC1GvZVlQ9Of3GP9y7X4vDpHshdlWotOnYKHaeu2d5cRVFHhEbrslkISgh/TRuyl7VIpnjOYUwMBpCiVH6M" +
+		"2lyDI6UR3Fbz4pVVAxGXnVhBExjBE=\n-----END CERTIFICATE-----"
```

Comment:
How was this generated? Does it need to be valid for this test to work?
Just asking because we don't seem to have needed it here before.

Reply:
copied over from unit tests, but yes we can probably live without it. I'll remove it.

Reply:
Removed.